### PR TITLE
[NETBEANS-5268] Alter Welcome Screen colours for readability

### PR DESCRIPTION
**Pull request by mhalachev**
https://issues.apache.org/jira/browse/NETBEANS-5268

I would like to suggest some small colour changes in the Welcome Screen.
Please refer to the issue link above and the discussion in #1826 for details.

### CODE REVIEW

**Comment by neilcsmith-net** [comment]
CC/ @DevCharly  - can't seem to add you as a reviewer yet?

**Comment by DevCharly** [comment]
@neilcsmith-net seems that I don't yet have the permissions.
Have emailed @geertjanw

**Comment by geertjanw** [comment]
I don't think there's anything I can do though, I believe you need to set up gitbox next.

**Comment by neilcsmith-net** [comment]
@DevCharly well, if you can add your thoughts on whether this seems good, we can merge. Thanks!

File: platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLightLaf.properties [hunk 1]
```diff
@@ -67,6 +67,10 @@ SlidingButton.attentionForeground=$ViewTab.attentionForeground
 PropSheet.setBackground=darken(@background,10%)
 PropSheet.setForeground=@foreground
 
+# Start Page
+nb.startpage.topbar.background=darken($TabbedPane.focusColor, 25%)
+nb.startpage.tab.border1.color=darken($TabbedPane.focusColor, 25%)
+nb.startpage.tab.border2.color=darken($TabbedPane.focusColor, 25%)
```
Comment (DevCharly) [inline platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLightLaf.properties:73]: Better use this to give tab borders same color as background:
~~~
nb.startpage.tab.border1.color=$nb.startpage.topbar.background
nb.startpage.tab.border2.color=$nb.startpage.topbar.background
~~~

**Review by DevCharly — CHANGES_REQUESTED** [review]

File: platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLightLaf.properties (hunk 1 above)
Comment (mhalachev) [reply platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatLightLaf.properties]: Thank you for your suggestion! I completely agree.

**Review by DevCharly — APPROVED** [review]

**Comment by mhalachev** [comment]
Thank you all!

**Comment by neilcsmith-net** [comment]
Thanks @mhalachev And, sorry, normally give people a heads-up before force pushing to their repo but needed to get in quickly. I see you deleted the branch anyway. :smile:

**Comment by mhalachev** [comment]
@neilcsmith-net It's all right, I got it! 😄

**Pull request merged**